permissions/Audio.permission

```diff
@@ -2,9 +2,16 @@
 
 # x-sailjail-translation-catalog = sailjail-permissions
 # x-sailjail-translation-key-description = permission-la-audio
-# x-sailjail-description = Play audio
+# x-sailjail-description = Play and record audio
```

Comment:
As this is what we show to user: IMO it needs to be in sync with reality -> having "and record" here is a good thing.

But it also means this description is not in sync with what README.md says and I'm wondering if the readme content should be expanded a bit to explain difference?

Reply:
That was my original point and I did erase the README.md part. Well, I'll add it back then.

Reply:
I guess it's okay as long as we clearly say that "ok, this gives you access to both (Audio = playback + recording) but you should really include also the other one (Microphone) if you need that functionality (recording)". IMO we should leave the technical bits of that out of the table because there just isn't a whole lot of space there.

Reply:
@spiiroin Added more descriptions to `README.md` with eae11d6 - what do you think, too much or just enough?

Reply:
I think it is ok now.